pkg/types/scanoptions.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+type ScanOptions struct {
+	VulnType []string
+}
+
+func IsVulnTypeSelected(scanOptions ScanOptions, vulnType string) bool {
```

Comment:
I think that IsVulnTypeSelected is unnecessary because this function is the same with `utils.StringInSlice`. Is there a reason to need this function?

Reply:
Did not know about `utils.StringInSlice`, I will delete IsVulnTypeSelected.

Reply:
Done